lua/lvim/lsp/manager.lua

```diff
@@ -31,6 +31,13 @@ local function resolve_config(name, user_config)
   local has_custom_provider, custom_config = pcall(require, "lvim/lsp/providers/" .. name)
   if has_custom_provider then
     Log:debug("Using custom configuration for requested server: " .. name)
+    local lsp_installer_servers = require "nvim-lsp-installer.servers"
+    local server_available, requested_server = lsp_installer_servers.get_server(name)
+    if server_available then
+      config.server = {
```

Comment:
I'm not really sure what sure what the use case is here? because if a server is available then all the options are passed to it anyway: https://github.com/LunarVim/LunarVim/blob/0481ec8dddf4bd8ed81c10ae98807fec97f6f872/lua/lvim/lsp/manager.lua#L105

what is `config.server`? this config table is the same as the one passed to the `setup()` function in lspconfig. I just tried with the latest commit on main branch for nvim-lsp-installer, and jsonls is still working as expected (at least with nlsp-settings).

Reply:
wait i think i fucked up 😅 

this is not needed
i was testing two branches at the same time and i though nlspsettings is not working

my bad 🤦

Reply:
I was just reading the comment from https://github.com/LunarVim/LunarVim/issues/2497#issuecomment-1106188017, and I realized that I haven't really tested if `:LspSettings rust-analyzer` would still work with rust-tools, so please test that 🥲 

_hopefully it's fine_

Reply:
<img width="1915" alt="Screen Shot 2022-04-22 at 6 25 35 PM" src="https://user-images.githubusercontent.com/10992695/164728787-f71ae04f-ee3b-42b8-8f97-038cbf4256f3.png">
😄